lua/lspconfig/racket_langserver.lua

```diff
@@ -0,0 +1,24 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.racket_langserver = {
+  default_config = {
+    cmd = {"racket", "--lib", "racket-langserver"};
+    filetypes = {"scheme"};
+    root_dir = util.path.dirname;
```

Comment:
There _is_ sometimes an `info.rkt`, the racket equivalent of node's package.json, but it isn't required.

A git root is a good idea, most racket projects use git.

Reply:
As far as I know though, I thought DrRacket only offers single file support. So no matter what this value is, it will also be offering only support for that individual file.

Reply:
If that's true, I still think we should be consistent with our root folder patterns because I'm planning to make some small assumptions about these in the future.

Reply:
Sure, I can make it `.git` then.

Reply:
However, if you do `.git` and there is not `.git` in the tree, then what does it fall back to?

Reply:
Just fallback to util.path.dirname, see this snippet from pyright.

```lua
root_dir = function(filename)
      return util.root_pattern(unpack(root_files))(filename) or
             util.path.dirname(filename)
end;
```